src/src/com/microsoft/aad/adal/StorageHelper.java

```diff
@@ -448,16 +448,6 @@ final synchronized private SecretKey getSecretKeyFromAndroidKeyStore() throws IO
                 throw new UnrecoverableKeyException("Couldn't find encrypted key in file");
             }
             
-            // Check if the retrieved keypair is empty. With the current limitation of 
-            // AndroidKeyStore, there is possibility that the alias is not wiped but 
-            // the key data is wiped, if this is the case, the retrieved keypair will
-            // be empty, and when we use the private key to do unwrap, we'll encounter 
-            // IllegalArgumentException
-            final PrivateKey privateKey = mKeyPair.getPrivate();
-            if (privateKey == null || privateKey.getEncoded() == null || privateKey.getEncoded().length == 0) {
```

Comment:
if it was ever null we would get InvalidKeyException("Need RSA private or public key"); 
But we've never had this crash